captum/optim/_core/output_hook.py

```diff
@@ -64,11 +39,15 @@ def forward_hook(module, input, output) -> None:
                     "As of 2019-11-22 please don't reuse nn.Modules in your models."
                 )
             if self.is_ready:
-                raise AbortForwardException("Forward hook called, all outputs saved.")
+                warn(
+                    "No outputs found from models. This can be ignored if you are "
+                    "optimizing on inputs only, without models. Otherwise, check "
+                    "that you are passing model layers in your losses."
+                )
```

Comment:
This warning seems to be displayed regardless of the targets used every time `collect_activations` is run:

```
/content/captum/captum/optim/_core/output_hook.py:43: UserWarning: No outputs found from models. This can be ignored if you are optimizing on inputs only, without models. Otherwise, check that you are passing model layers in your losses.
  "No outputs found from models. This can be ignored if you are "
```

Reply:
Fixed this with appropriate warning suppression.